acceptance/tests/resource/service/ticket_3927_redhat_systemd_vs_service_provider.rb

```diff
@@ -0,0 +1,46 @@
+test_name 'RedHat Service vs. Systemd Provider Validation'
+
+# A simple acceptance test to ensure basic usage of the service
+# provider works for a mix of sysV and systemd RedHat Linux platforms
+confine :to, :platform => /el|centos|fedora/
+
+manifest_install_httpd = %Q{
+  package { 'httpd':
+    ensure => present,
+  }
+}
+
+manifest_httpd_enabled = %Q{
+  service { 'httpd':
+    enable => true,
+  }
+}
+
+manifest_httpd_disabled = %Q{
+  service { 'httpd':
+    enable => false,
+  }
+}
+
+agents.each do |agent|
+  distro = on(agent, facter('operatingsystem')).stdout.chomp
+  if distro == 'fedora'
+    majrelease = on(agent, facter('operatingsystemmajrelease')).stdout.chomp.to_i
+    if majrelease < 17
+      skip_test "Test not applicable to Fedora #{majrelease}"
+    elsif majrelease > 20
```

Comment:
@peterhuene I tried testing F21 with our internal acceptance test beaker setup and we don't offer a repo file to set it up. (It's looking in http://builds.puppetlabs.lan/puppet/3.7.3/repo_configs/rpm/). So I assumed I needed to skip F21 to avoid causing errors in our CI testing. Let me know if that assumption is wrong.